examples/siro_sandbox/episode_helper.py

```diff
@@ -2,12 +2,17 @@ class EpisodeHelper:
     def __init__(self, habitat_env):
         self._habitat_env = habitat_env
 
-        self._num_iter_episodes: int = len(self._habitat_env.episode_iterator.episodes)  # type: ignore
+        self._episode_iterator_cycle = habitat_env.episode_iterator.cycle
+        self._num_iter_episodes: int = len(habitat_env.episode_iterator.episodes)  # type: ignore
         self._num_episodes_done: int = 0
 
     @property
     def num_iter_episodes(self):
-        return self._num_iter_episodes
+        return (
+            self._num_iter_episodes
+            if not self._episode_iterator_cycle
+            else float("inf")
```

Comment:
-1 would make more sense than a float here.

Reply:
In other app states we display num episodes left help text. It outputs num_iter_episodes - num_episodes_done. If we set -1 then we will be getting negative number,  with `float("inf")` inf will be displayed. IMHO it is more meaningful.